pyomo/contrib/mindtpy/tests/nonconvex1.py

```diff
@@ -0,0 +1,24 @@
+from pyomo.environ import *
+
```

Comment:
We need to add the reference to the paper where this was taken from. Same case with all the other examples.